### PR TITLE
Set project label on branch creation

### PR DESCRIPTION
**Pull request by macanton**
Current PR adds the possibility to add labels to Jira issue and also sets 'repo-XXX' label to issue on branch checkout.

## Check
- checkout branch
- run `go build;mv got /usr/local/bin/`
- check that on branch checkout git `got -b XXXX` repo label set
- check that on jira issue creation `got -cj` repo  label set

### CODE REVIEW

File: pkg/jira/api.go [hunk 1]
```diff
@@ -120,6 +120,47 @@ func CreateIssue(summary string) (string, error) {
...
+	requestURL, err := getRequestURL(jiraOperationUpdateIssue, issueKey)
+	if err != nil {
+		return labels, err
+	}
+
+	var updateIssueLabels []UpdateIssueLabels
+	for _, label := range labels {
+		updateIssueLabels = append(updateIssueLabels, UpdateIssueLabels{Add: label}) // note the = instead of :=
+	}
+	formValues := UpdateIssueData{
```
Comment (Den1k22) [inline pkg/jira/api.go:136]: Do we need extra space after `UpdateIssueData` and before bracket?

File: pkg/jira/api.go [hunk 2]
```diff
@@ -120,6 +120,47 @@ func CreateIssue(summary string) (string, error) {
...
+		Update: UpdateIssueDataFields{
```
Comment (Den1k22) [inline pkg/jira/api.go:137]: Same? :)

File: main.go [hunk 3]
```diff
@@ -89,9 +102,35 @@ func createJiraTicketAndCheckBranch() {
...
+	}
+
+	issueKey := issueKeys[0]
+	newLabels, err := jira.AddIssueLabels(issueKey, config.Options.Labels)
+	if err != nil {
+		printErrorToConsole(err)
+		return
+	}
+
+	printInfoToConsole(fmt.Sprintf("Jira issue lables updated to '%s'", strings.Join(newLabels, ", ")))
```
Comment (Den1k22) [inline main.go]: Typo: lables ->  labels :)

**Review by Den1k22 — CHANGES_REQUESTED** [review]

File: pkg/jira/api.go (hunk 1 above)
Comment (macanton) [reply pkg/jira/api.go:136]: Good question :) As I can see for constructors there is no need for empty line before brackets :)

File: pkg/jira/api.go (hunk 2 above)
Comment (macanton) [reply pkg/jira/api.go:137]: Same :)

**Review by Den1k22 — APPROVED** [review]

**Pull request merged**